packages/ui-editable/src/InPlaceEdit/theme.js

```diff
@@ -22,8 +22,19 @@
  * SOFTWARE.
  */
 
-export default function generator({ typography }) {
+/**
+ * Generates the theme object for the component from the theme and provided additional information
+ * @param  {Object} theme The actual theme object.
+ * @param  {Object} themeOverride User provided overrides of the default theme mapping.
+ * @return {Object} The final theme object with the overrides and component variables
+ */
+const generateComponentTheme = (theme, themeOverride = {}) => {
+  const { typography } = theme
+
   return {
-    fontFamily: typography.fontFamily
+    fontFamily: typography?.fontFamily,
```

Comment:
This one I'm not sure about, because it was the "themeable" prop of FocusView, but not sure we want to make it overridable for InPlaceEdit too. I mean the only usecase of this would be that once you click on the Text to edit, it would change font family...

Edit: I tried to set it with `themeOverride={{fontFamily: "serif"}}` on the component, but it didn't seem to have any effect, since the text child inside had it's own font family anyway. So I'd say this is only confusing to list it as an overridable variable, but has no effect.

Reply:
good catch, I'll delete it